ui/Screen/Settings.svelte

```diff
@@ -1,31 +1,46 @@
 <script>
+  import {
+    clear,
+    clearCache,
+    settings,
+    updateAppearance,
+    updateRegistry,
+  } from "../src/session.ts";
+
   import { Title, Text, Button } from "../DesignSystem/Primitive";
   import { SidebarLayout, SegmentedControl } from "../DesignSystem/Component";
 
-  const themeOptions = [
-    {
-      title: "Light",
-      value: 0,
-    },
-    {
-      title: "Dark",
-      value: 1,
-    },
-  ];
   const networkOptions = [
     {
       title: "Emulator",
-      value: 0,
+      value: "emulator",
     },
     {
       title: "FFnet",
-      value: 1,
+      value: "ffnet",
     },
     {
       title: "Testnet",
-      value: 2,
+      value: "testnet",
+    },
+  ];
+
+  const updateNetwork = (event) =>
+    updateRegistry({ ...$settings.registry, network: event.detail });
+
+  const themeOptions = [
+    {
+      title: "Light",
+      value: "light",
+    },
+    {
+      title: "Dark",
+      value: "dark",
     },
   ];
+
+  const updateTheme = (event) =>
+    updateAppearance({ ...$settings.apperane, theme: event.detail });
```

Comment:
Typo: apperane -> appearance.

Reply:
How does this even work with the typo?

Reply:
It works because there is no other field in `Appearance`, it is here to account for future extensions to that object.